shark-graph/src/main/java/shark/HprofHeapGraph.kt

```diff
@@ -132,23 +132,23 @@ class HprofHeapGraph internal constructor(
     return index.objectIdIsIndexed(objectId)
   }
 
-  internal fun fieldName(
+  override fun fieldName(
     classId: Long,
-    fieldRecord: FieldRecord
+    nameStringId: Long
```

Comment:
This is some weird optimization, but I noticed that having a reference to `FieldRecord` in the code generates an additional check for `null` and takes more CPU time than having a simple `long`

Reply:
Is this public API? If so, I should revert it

Reply:
It wasn't a public API before, but it looks like now you're making it public?

Reply:
How much "more CPU time" exactly is this? Really not great to expose this API with just longs, easy to make mistakes.